src/main/java/org/jenkinsci/plugins/fodupload/steps/FortifyStaticAssessment.java

```diff
@@ -200,9 +202,11 @@ public String getFunctionName() {
         // Form validation
         @SuppressWarnings({"ThrowableResultOfMethodCallIgnored", "unused"})
         @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
+        @POST
         public FormValidation doTestPersonalAccessTokenConnection(@QueryParameter(SharedUploadBuildStep.USERNAME) final String username,
                                                                   @QueryParameter(SharedUploadBuildStep.PERSONAL_ACCESS_TOKEN) final String personalAccessToken,
                                                                   @QueryParameter(SharedUploadBuildStep.TENANT_ID) final String tenantId) {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```

Comment:
Isn't Adminster the most permissive role in Jenkins?  Won't this affect most users of the plugin?  I'd assume not everyone using this is intended to have access to global settings in Jenkins, but I could be wrong.